pkg/apis/nmstate/v1/state_test.go

```diff
@@ -0,0 +1,103 @@
+package v1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	yaml "github.com/ghodss/yaml"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	var (
+		obtainedStruct   NodeNetworkState
```

Comment:
nodeNetworkStateStruct

Reply:
Done, I like obtained/expected idiom at unit test, but since this is unmarshal it's implicit.